docroot/themes/contrib/civictheme/civictheme_library/components/00-base/grid/_index.scss

```diff
@@ -122,8 +122,10 @@
 }
 
 .ct-justify-content-end {
-  display: flex;
-  justify-content: flex-end;
+  @include ct-breakpoint(m) {
```

Comment:
This class used in header was overwriting components display. This needed only needed on desktop